.github/workflows/temsimulator.yml

```diff
@@ -0,0 +1,48 @@
+name: "Building TEM simulator container"
+
+on:
+  push:
+    branches: [ master, temsim-container ]
+
+  pull_request:
+    branch: [ master, temsim-container ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check Out Repo
+        uses: actions/checkout@v2
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
```

Comment:
what and were is Docker Hub?

Reply:
I gave my credentials so the image is being built and pushed here: https://hub.docker.com/layers/174436511/fpoitevi/simspi-temsimulator/latest/images/sha256-2b0366b0d6f6d764f99b336a50e9bbf2f64e707fe49769d0685672484882ed08?context=repo